app/graphql/inputs/digital_object/order_by_input.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Inputs
+  class DigitalObject::OrderByInput < Types::BaseInputObject
+    description 'Digital Object Sort Parameters'
+
+    argument :field, Enums::DigitalObject::OrderFieldsEnum, required: false, default_value: 'score'
```

Comment:
are the default_value's here not supposed to be the enum values?

Reply:
That's what I thought at first, but its supposed to be the actual value the enum is representing.